fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -198,7 +198,7 @@ open class DefaultToolbarMenu(
      */
     @VisibleForTesting(otherwise = PRIVATE)
     fun shouldShowTranslations(): Boolean = selectedSession?.let {
```

Comment:
Thanks for adding this check! Can you also block the translations entry point on the [BrowserMenu](https://searchfox.org/mozilla-mobile/rev/c187a25712a5a5bb196c552ed6c14c1df0e3e493/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt#303) too?

Reply:
Done